configure.ac

```diff
@@ -14,19 +17,49 @@ AS_IF([test "x$with_systemdsystemunitdir" != "xno"], [
   AC_SUBST([systemdsystemunitdir], [$with_systemdsystemunitdir])
 ])
 
+# build updater with metrics
+AC_ARG_ENABLE([metrics],
+              AS_HELP_STRING([--disable-metrics], [Whether to build eos-updater with metrics support]),
+              [want_metrics=$withval],
+              [want_metrics=yes])
+AM_CONDITIONAL([WITH_METRICS], [test "x$want_metrics" = 'xyes'])
+
+# port used for eos-server
+AC_ARG_WITH([server-port],
+            AS_HELP_STRING([--with-server-port], [Port number for the update server]),
+            [port=$withval],
+            [port=12345])
```

Comment:
Seems fishy to me to make it a configure option (or at least we could have a configure-default and make it runtime configurable).